packages/odyssey-storybook/src/components/odyssey-mui/PasswordField/PasswordField.stories.tsx

```diff
@@ -85,7 +85,7 @@ export const Default: StoryObj<PasswordFieldProps> = {
       expect(fieldElement.type).toBe("password");
 
       const buttonElement = canvasElement.querySelector(
-        '[aria-label="toggle password visibility"]'
+        '[aria-label="Show password"]'
```

Comment:
👍

Reply:
Addressed + added test to make sure labels toggle as we expect